### PR TITLE
feat: add Directus support

### PR DESCRIPTION
**Pull request by danestves**
Internally at Kazel we use a custom transformer to display the images of our backend to our frontend, I moved all the code to here since there is an issue open about it, added tests and an example image which you can test.

Closes https://github.com/ascorbic/unpic/issues/36

### CODE REVIEW

File: data/paths.json [hunk 1]
```diff
@@ -3,5 +3,6 @@
   "/_next/image": "nextjs",
   "/_next/static": "nextjs",
   "/_vercel/image": "vercel",
-  "/is/image": "scene7"
+  "/is/image": "scene7",
+  "/assets": "directus"
```
Comment (ascorbic) [inline data/paths.json:7]: I'm concerned that this is a bit too generic to use as a path. There could be a lot of false positives. We might need to stick with manually specifying the CDN

Comment (danestves) [reply data/paths.json]: Done! I have removed that code to avoid false positives

Comment (ascorbic) [reply data/paths.json]: Thanks

**Review by ascorbic — COMMENTED** [review]
Thanks! I wanted to add support for manually setting the CDN to the playground so I coiuld tests this there. I did that today and it works great!

**Comment by ascorbic** [comment]
@danestves the sample URL that you provided here is no longer resolving, menaing the Directus tests no longer pass. Would you have a stable sample image URL it could be switched for?

**Pull request merged**